config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
+  root 'static_pages#home'
+  match "/help",to: "static_pages#help", via: "get"
+  match "/about",to: "static_pages#about", via:"get"
+  match "/contact", to: "static_pages#contact", via: "get"
+  match "/signup", to: "users#new", via: "get"
```

Comment:
sao dùng cú pháp này e
dùng cú pháp này nhé: get "/contact", to:  "static_pages#contact"